components/book/BookCard/BookCard.tsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react'
+import Image from 'next/image'
+import { Book } from '../../../types'
 
-const BookCard = () => {
-  return <div>This is a book card.</div>
+const BookCard = ({ book }: { book: Book; eager: boolean }) => {
+  return (
+    <div>
+      <Image
+        alt="book picture"
```

Comment:
should be something meaningfull like `book.title`

Reply:
good one, will be updated in the next PR